api/helm/api/values.yaml

```diff
@@ -29,7 +29,7 @@ varnish:
 
 ingress:
   annotations:
-    kubernetes.io/ingress.global-static-ip-name: api-platform-demo-ip
+    kubernetes.io/ingress.global-static-ip-name: demo-ip
     # kubernetes.io/ingress.class: gce
```

Comment:
Remove commented code.

Reply:
Out of this PR scope. This PR fixes the deploy, it does not improve/clean it. This is supposed to be done in [your PR](https://github.com/api-platform/demo/pull/61)